CMSgov/design-system
# [EDUCATE-380] Add HCgov specific Accordion

https://jira.cms.gov/browse/EDUCATE-380

## Summary

Add HCgov specific Accordion styles to the design system.
- Applications using Core and Medicare design system will default to the existing Accordion styles
- Applications using Healthcare design system will default to this alternate Accordion styles

### Changed

- Add `openIcon` and `closeIcon` props with default icons on the core Accordion component. 
- On HCgov design system:
  - override the default core icons with HCgov Accordion icons
  - override the core Accordion scss
  - add storybook for the specific HCgov Accordion
  - add the specific HCgov Accordion to the doc site

## TODOs

- [x] How to verify that the new HCgov default props are being applied for applications using the HCgov accordion? (*followed Anna's suggestion to use the `react-app` in the `examples` directory of the DS repo for verification*)
- [x] Who is responsible for updating the guidelines on HCgov Accordion doc site (_Accordion.doc.scss)?

## How to test

- Verify that the Accordion component on core and medicare design system remains unchanged
- Verify that the Accordion component on Hcgov design system is updated with the Hcgov specific Accordion styles
